valohai_yaml/objs/parameter_widget.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional, Union
+
+from valohai_yaml.objs.base import Item
+from valohai_yaml.types import SerializedDict
+
+
+class ParameterWidget(Item):
+    """An UI widget for editing parameter values."""
+
+    def __init__(
+        self,
+        *,
+        type: str,
+        settings: Optional[dict] = None,
+    ) -> None:
+        self.type = type.lower()
+        self.settings = settings
```

Comment:
Might be nicer that `self.settings` always was a thing, even if empty.

```suggestion
        self.settings = dict(settings or {})
```

This doesn't need changes to the serializer though since empty dicts are falsy ✨